charmcraft/commands/init.py

```diff
@@ -72,11 +76,20 @@ def run(self, args):
         if not re.match(r"[a-z][a-z0-9-]*[a-z0-9]$", args.name):
             raise CommandError("{} is not a valid charm name".format(args.name))
 
+        if not args.series:
+            try:
+                os_release = parse_os_release()
+                # TODO: support non-Ubuntu things
+                args.series = [os_release["UBUNTU_CODENAME"]]
+            except (UnicodeDecodeError, KeyError):
```

Comment:
When you can have UnicodeDecodeError?

Reply:
`os-release` could be garbage?
I'm dropping this entirely to default to `"kubernetes"` fwiw